### PR TITLE
mkosi: add missing build dependencies

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: mkosi.default [hunk 1]
```diff
@@ -16,8 +16,10 @@ BuildPackages=
         gcc
         libacl-devel
         libcurl-devel
+        libzstd-devel
         openssl-devel
         meson
+        python2-sphinx
```
Comment (fsateler) [inline mkosi.default]: python2?

**Comment by vbatts** [comment]
it's what provides `/usr/bin/sphinx-build`

**Comment by ignatenkobrain** [comment]
@vbatts then why not `/usr/bin/sphinx-build` ?

**Comment by vbatts** [comment]
good point. updated.

**Pull request merged**